scene/animation/animation_node_state_machine.cpp

```diff
@@ -317,7 +317,7 @@ float AnimationNodeStateMachinePlayback::process(AnimationNodeStateMachine *p_st
 					// stopped, invalid state
 					String node_name = start_request;
 					start_request = StringName(); //clear start request
-					ERR_FAIL_V_MSG(0, "Can't travel to '" + node_name + "' if state machine is not playing.");
+					ERR_FAIL_V_MSG(0, "Can't travel to '" + node_name + "' if state machine is not playing. Maybe you need to set auto-start for one of the nodes in your state machine?");
```

Comment:
Use the exact property name and capitalize it, so that it can be searched more easily in the class reference:

```suggestion
					ERR_FAIL_V_MSG(0, "Can't travel to '" + node_name + "' if state machine is not playing. Maybe you need to enable Autostart for one of the nodes in your state machine?");
```

Reply:
Updated -- I couldn't find "Autostart" referenced in the documentation (I'll do a separate PR to update to include a screenshot with it in the AnimationTree tutorial), so I also included a hint to call `.start()` on the state machine.

Reply:
Nvm, found a reference to "Autoplay on Load" in the [Animations tutorial](https://docs.godotengine.org/en/stable/getting_started/step_by_step/animations.html?highlight=autoplay#editing-the-animation).

Reply:
PR to update AnimationTree tutorial: https://github.com/godotengine/godot-docs/pull/4652